checkov/terraform/checks/resource/gcp/GoogleSubnetworkPrivateGoogleEnabled.py

```diff
@@ -5,6 +5,8 @@
 from checkov.common.models.enums import CheckCategories, CheckResult
 from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
 
+# private_ip_google_access is not supported for subnetworks with the following purpose set
+PURPOSE_EXCEPTIONS = ["INTERNAL_HTTPS_LOAD_BALANCER", "REGIONAL_MANAGED_PROXY", "GLOBAL_MANAGED_PROXY"]
```

Comment:
```suggestion
PURPOSE_EXCEPTIONS = ["INTERNAL_HTTPS_LOAD_BALANCER", "REGIONAL_MANAGED_PROXY", "GLOBAL_MANAGED_PROXY", "PRIVATE_SERVICE_CONNECT","PRIVATE_NAT"]
```
I believe these should also be excluded